### PR TITLE
MPC picklist group question export changes and some minor study config changes

### PR DESCRIPTION
**Pull request by ssettipalli**
changes to support picklist Group question in DataExport (both csv and elastic)
some minor study config and migration changes.

### CODE REVIEW

File: pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/PicklistQuestionFormatStrategy.java [hunk 1]
```diff
@@ -50,6 +60,28 @@
...
+                stableIdTxt.put("optionStableId", optionDef.getStableId());
+                stableIdTxt.put("optionText", HtmlConverter.getPlainText(optionDef.getOptionLabelTemplate().render("en")));
+                options.add(stableIdTxt);
+            }
+            groupDef.put("groupStableId", groupStableId);
+            groupDef.put("groupText", groupTxt);
+            groupDef.put("options", options);
+            groups.add(groupDef);
+        }
+        if (!groups.isEmpty()) {
```
Comment (ssettipalli) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/PicklistQuestionFormatStrategy.java:81]: Makes "groups" optional element in elastic json. Will exist only for PL Groups.
wondered if it should be empty list element for non-PL Group questions like groups: []

File: elasticsearch/templates/activity_definition.json [hunk 2]
```diff
@@ -41,6 +41,26 @@
             "dateFields": {
               "type": "keyword"
             },
+            "groups": {
```
Comment (yufengwng) [inline elasticsearch/templates/activity_definition.json:44]: So when we do a release, we'll need to update the template in ES and re-upload all the activity definitions?

**Review by yufengwng — APPROVED** [review]
👍

File: elasticsearch/templates/activity_definition.json (hunk 2 above)
Comment (ssettipalli) [reply elasticsearch/templates/activity_definition.json:44]: yes yufeng

**Pull request merged**